Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.    
</pre><p>Can you also add the beams to both plots with: <a href="https://docs.astropy.org/en/stable/api/astropy.visualization.wcsaxes.add_beam.html#astropy.visualization.wcsaxes.add_beam" target="_blank">https://docs.astropy.org/en/stable/api/astropy.visualization.wcsaxes.add_beam.html#astropy.visualization.wcsaxes.add_beam</a>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>